pegasus/sites.v3/code.org/public/educate/it.md

```diff
@@ -42,7 +41,13 @@ For the very best experience with all Code.org content, we recommend consulting
 | **To use Web Lab** | `https://downloads.computinginthecore.org`<br/>`https://codeprojects.org`                                                                              |
 
 ## Mobile Support Details
-Our Hour of Code activities and CS Fundamentals courses can be used on mobile devices. While these activities should be usable, they are not optimized for a mobile experience and have some known limitations (e.g. touch targets may be small, accessory elements may display offscreen, and text may be difficult to input using the on-screen keyboard). Note: the CS Discoveries, CS Principles, and Intro to App Lab courses are not usable on mobile devices.
+All CS Discoveries, CS Principles, and Intro to App Lab courses are not compatible with tablets or mobile devices - a full keyboard and mouse (or touchpad, like on Macbooks) are required. Surface Pros with a keyboard attachment and Chromebooks are compatible. iPads or other tablets with a keyboard attachment are not compatible.
+
+All other content, such as K-5 courses (Computer Science Fundamentals) and Hour of Code activities are compatible on iPads and Chromebooks in addition to all other modern laptop or desktop computers.
+
+### Mobile Phones
+
+Our Hour of Code activities and CS Fundamentals courses can be used on mobile phones. While these activities should be usable, they are not optimized for a mobile phone experience and have some known limitations (e.g. touch targets may be small, accessory elements may display offscreen, and text may be difficult to input using the on-screen keyboard). The CS Discoveries, CS Principles, and Intro to App Lab courses are not usable on mobile phones.
```

Comment:
I think we should have a stronger stance about the usability. Can we replace 'should be usable' with "are usable" or "While these activities can be completed on mobile phones, they are not . . .".

Reply:
What do you think of this?
"Our Hour of Code activities and CS Fundamentals courses can be completed on mobile phones. While these activities are be usable, they are not optimized..."

Reply:
I like it! (But I think there's a typo with 'are be').

Reply:
Haha thanks for catching that